packages/utils/src/strings.js

```diff
@@ -1,17 +1,14 @@
-export { default as kebabCase } from 'lodash.kebabcase';
-export { default as capitalize } from 'lodash.capitalize';
-export { default as upperFirst } from 'lodash.upperfirst';
+import _ from 'lodash';
+
+export const kebabCase = _.kebabCase;
+export const capitalize = _.capitalize;
+export const upperFirst = _.upperFirst;
+export const repeat = _.repeat;
```

Comment:
Using the `lodash.whatever` exports (a) doesn't get us anything and (b) prevents lodash's types from filtering through as the types are only defined for the actual `lodash` package.

Reply:
I was going to comment on this 🙂  I'd like to stop re-exporting these completely and use them directly as needed.

Reply:
Middle ground would be like `export { kebabCase } from 'lodash'`, but that will probably involve fiddling with `allowSyntheticDefaultImports` or other config.

Reply:
> I'd like to stop re-exporting these completely and use them directly as needed

Just to confirm! For example, a file within the components package, would we do:

```
import { kebabCase } from 'lodash'
```

---

For some context, I wasn't sure what the (long term) stance was for `lodash`, which is why I initially kept it's usage minimal with the G2 Components development.

If we're certain lodash will be with us for a while, then let's just use it :).

Reply:
> Middle ground would be like export { kebabCase } from 'lodash', but that will probably involve fiddling with allowSyntheticDefaultImports or other config.

Correct, that doesn't work with the current config 🤔 

> Just to confirm! For example, a file within the components package, would we do:

Yes, that is preferred to re-exporting everything.

> If we're certain lodash will be with us for a while, then let's just use it :).

Given `lodash` is used extensively in Gutenberg it's probably safe to say it's going to stick around 🙂 🙃